Modern/wrapperPlugins/src/main/java/org/bonej/wrapperPlugins/ThicknessWrapper.java

```diff
@@ -143,12 +144,11 @@ public void run() {
 		}
 	}
 
-	private void addMapResults(final ImagePlus map) {
+	private void addMapResults(String label, final ImagePlus map) {
 		if (map == null) {
```

Comment:
The parameter isn't necessary. Because `inputImage` is a member of the class, this non-static method can get the label by calling `inputImage.getTitle()`